examples/SampleUtils.cpp

```diff
@@ -45,10 +45,10 @@ static CmdBufType cmdBufType = CmdBufType::Terrible;
 #else
 static CmdBufType cmdBufType = CmdBufType::None;
 #endif  // defined(WEBNN_ENABLE_WIRE)
-static webnn::wire::WireServer* wireServer = nullptr;
-static webnn::wire::WireClient* wireClient = nullptr;
-static utils::TerribleCommandBuffer* c2sBuf = nullptr;
-static utils::TerribleCommandBuffer* s2cBuf = nullptr;
+static std::unique_ptr<webnn::wire::WireServer> wireServer = nullptr;
```

Comment:
WDYT about moving these into a new "wire helper" object?

The problem with static objects is they themselves do not show up as leaks, only the contents, the sub-objects within, in-turn that make allocations or call `new`.

Reply:
Sorry, I haven't understood, could you explain more clearly? Do you mean move into a struct/class like 
```
WireHelper
{static std::unique_ptr<webnn::wire::WireServer> wireServer = nullptr;
static std::unique_ptr<webnn::wire::WireClient> wireClient = nullptr;
static std::unique_ptr<utils::TerribleCommandBuffer> c2sBuf = nullptr;
static std::unique_ptr<utils::TerribleCommandBuffer> s2cBuf = nullptr;}
```? And what is the difference compared with current style?
@bbernhar

Reply:
@mingmingtasd Yup, if we move those variables into a `WireHelper` class, as **non-static** members, and the `WireHelper` leaks again, those sub-objects should appear in the leak report as you expect.